homeassistant/components/mill/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "title": "Mill",
```

Comment:
We don't have to add brand names as title to the translation files. It will fall back to the integration name from the manifest.